### PR TITLE
Loki + promtail

### PR DESCRIPTION
**Pull request by kucerakk**
- Loki + Promtail helm chart 
 - Loki role to add auth/tls/ingress
 - Promtail role for VM log gathering
 - Inventory update

### CODE REVIEW

File: playbooks/roles/promtail/templates/promtail.service.j2 [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+[Service]
+Restart=on-failure
+
+ExecStartPre=/usr/bin/rm -f /%t/%n-pid /%t/%n-cid
+
+ExecStart=/usr/bin/podman run \
+          --conmon-pidfile /%t/%n-pid \
+          --cidfile /%t/%n-cid \
+          --cgroups=no-conmon -d \
```
Comment (gtema) [inline playbooks/roles/promtail/templates/promtail.service.j2:16]: we usually also have `--log-opt=path=/dev/null` for podman. Should we have it here?

**Review by gtema — APPROVED** [review]
small nit inside

File: playbooks/roles/promtail/templates/promtail.service.j2 (hunk 1 above)
Comment (kucerakk) [reply playbooks/roles/promtail/templates/promtail.service.j2:16]: right, it is good to have it...changed in the latest commit

**Review by gtema — APPROVED** [review]

**Review by gtema — APPROVED** [review]

**Comment by gtema** [comment]
regate

**Pull request merged**